### DISCUSSION TITLE
Firmware v2 / MP3 Wav format / Battery

### DISCUSSION THREAD

**Discussion by fredcazaux** [marian-m12l/studio#312] [Q&A]
Je vais commencer cette discussion en anglais pour la communauté. Mais je peux traduire en français si nécessaire.

Since I updated the firmware to v2, I noticed 2 changes : 

1. The battery consumes really faster !
2. The volume seems to be a bit lower with this firmware.

I've read here and there that the MP3 format would be the reason for the battery to discharge faster. So first questions : 

- [ ] **Q1 : Is the MP3 format the culprit of fast battery discharging ?**
- [ ] **Q2 @marian-m12l  : Is there a way to use WAV on v2 as on v1 ?**
I've tried using WAV 32000Hz Mono in the STUDIO editor, but it seems that it is automatically converted to MP3 before transferring...

Concerning the volume : 

- [ ] **Q3 : Is there a reference for loudness to use in audio files ?** 

If nobody knows that, here is what I've found : 
I opened several original Lunii audio files on the FS (Suzanne et Gaston), and using Adobe Audition, here is the loudness statistics I've found : 

- ITU Loudness : -13 LUFS
- Total RMS : ~= -14.5dB
- Loudness : ~= -11dB
- Perceived : ~= -8dB

To get an audio that matches this and has no distortion and a clean sound, I've used, in the order : 

1. Favorites > Convert to Mono
2. Effects > Amplitude and Compression > DeEsser... > Female DeEsser preset
3. Effects > Amplitude and Compression > DeEsser... > Female DeSher preset
4. Effects > Amplitude and Compression > Dynamics > AutoGate On + AutoGate Threshold -40dB + Compressor ON + Compressor Threshold -3dB + Compressor Ratio 3
5. Effects > Match Loudness > Match To : ITU-R BS.1770-3 Loudness + Target Loudness -13LUFS + Tolerance 0.5 LU + Max True Peak -0.1dBTP

Hope this will help....

**Comment by mattmofr** [comment]
> Je vais commencer cette discussion en anglais pour la communauté. Mais je peux traduire en français si nécessaire.
> 
> Since I updated the firmware to v2, I noticed 2 changes :
> 
> 1. The battery consumes really faster !
> 2. The volume seems to be a bit lower with this firmware.
> 
> I've read here and there that the MP3 format would be the reason for the battery to discharge faster. So first questions :
> 
> * [ ]  **Q1 : Is the MP3 format the culprit of fast battery discharging ?**
> * [ ]  **Q2 : Is there a way to use WAV on v2 as on v1 ?**
>   I've tried using WAV 32000Hz Mono in the STUDIO editor, but it seems that it is automatically converted to MP3 before transferring...
> 
> Concerning the volume :
> 
> * [ ]  **Q3 : Is there a reference for loudness to use in audio files ?**
> 
> If nobody knows that, here is what I've found : I opened several original Lunii audio files on the FS (Suzanne et Gaston), and using Adobe Audition, here is the loudness statistics I've found :
> 
> * ITU Loudness : -13 LUFS
> * Total RMS : ~= -14.5dB
> * Loudness : ~= -11dB
> * Perceived : ~= -8dB
> 
> To get an audio that matches this and has no distortion and a clean sound, I've used, in the order :
> 
> 1. Favorites > Convert to Mono
> 2. Effects > Amplitude and Compression > DeEsser... > Female DeEsser preset
> 3. Effects > Amplitude and Compression > DeEsser... > Female DeSher preset
> 4. Effects > Amplitude and Compression > Dynamics > AutoGate On + AutoGate Threshold -40dB + Compressor ON + Compressor Threshold -3dB + Compressor Ratio 3
> 5. Effects > Match Loudness > Match To : ITU-R BS.1770-3 Loudness + Target Loudness -13LUFS + Tolerance 0.5 LU + Max True Peak -0.1dBTP
> 
> Hope this will help....

Thanks for the accoustic analysis and audio effects channel suggestion. I guess the channel needs to be adapted depending on the original audio file (male telling story, music or not, etc.) ? If you record your own story you will need to adapt your audio treatment wether you use studio dynamic, ribbon or condenser mics or even more if you use your computer integrated microphone, don't you think ?

**Reply by fredcazaux** [reply]
Actually, not that much. Those effects are more intended to match the Lunii specifications and imperfections than improving sound properly speaking. The Lunii is Mono, so it's better to 1st convert any audio in Mono on purpose instead of letting a script doing it at the end. The two DeEssers (with female presets) attenuate frequencies that are especially increased by the Lunii speaker, so nothing to do with the narrator being a man or a woman. You're right concerning the AutoGate, it should be applied only on stories, not on music.... For the Compressor, with a threshold at -3dB, it's intended to attenuate small clippings. I don't use it to apply dynamics on the actual content... And finally, the Math Loudness is based on Lunii stories. So yes, those parameters are good for story telling. They may need adjustments for music (I'll give it a try soon...)

And to be honest, I've applied all those modifications on stories that sounded good on firmware v1 but horrible on v2. I don't know if it's only the MP3 decoding which is really poor on v2, or if the conversion implies to be more prudent. I had several stories recorded with my phone headphones mic. They sounded quite good as it was on v1. But when updating to v2 and uploading again, they sounded horrible with lots of saturation... 

You have all the explanations !

**Reply by mattmofr** [reply]
Thanks for your detailed explanation. That's awesome. I'll apply those settings from now on !
I have the feeling that you're keen on "MAO" ;)

**Comment by benjapan** [comment]
Thanks a lot for this work on Lunii. 
about Q3, I’ve noticed the low sound level of imported MP3. 
i checked native MP3 files with audacity, Lunii audio files are a lot compressed!
The only way to have same sound level with our MP3 is to damage the file with strong gain increase. I prefer use native MP3 and change lunii headsets that are not so well made.

**Reply by fredcazaux** [reply]
Yes I've seen that their audio files are very much compressed to gain level. I found that using the "Match Loudness" set to -13 LUFS in Adobe Audition does the job !
